src/game.cpp

```diff
@@ -411,7 +411,7 @@ void game::init_ui()
         locW = sidebarWidth;
         stat2H = 2;
         stat2W = sidebarWidth;
-        messH = TERMY - (statH + locH + stat2H);
+        messH = TERRAIN_WINDOW_HEIGHT - (statH + locH + stat2H);
```

Comment:
After rooting around a bit, I found there's even a variable there for this, TERRAIN_WINDOW_TERM_HEIGHT, testing now.